src/main/java/it/pagopa/ecommerce/payment/methods/controller/PaymentMethodsController.java

```diff
@@ -289,4 +296,53 @@ private Mono<String> getAuthenticationToken(ServerWebExchange exchange) {
                         .map(header -> header.substring("Bearer ".length()))
         );
     }
+
+    @Warmup
+    public void getAllPaymentMethodsWarmupMethod() {
+        WebClient
+                .create()
+                .get()
+                .uri("http://localhost:8080/payment-methods")
+                .header(X_CLIENT_ID, PaymentMethodRequestDto.ClientIdEnum.CHECKOUT.toString())
+                .retrieve()
+                .bodyToMono(PaymentMethodsResponseDto.class)
+                .block(Duration.ofSeconds(30));
+    }
+
+    @Warmup
+    public void calculateFeesWarmupMethod() {
```

Comment:
could you please add this warmup method also for v2 controller too?

Reply:
Done.